example.json

```
@@ -0,0 +1,52 @@
+{
```

This should be a list and not a dictionary. Dictionaries are not ordered by default.

```suggestion
[
```